applicationconfig/client/src/test/java/com/azure/applicationconfig/ConfigurationAsyncClientTest.java

```diff
@@ -618,6 +619,34 @@ public void listRevisions() {
                 .verifyComplete();
     }
 
+    /**
+     * Verifies that the range header for revision selections returns the expected values.
+     */
+    @Test
+    public void listRevisionsWithRange() {
```

Comment:
Is there a test for 416? (RangeNotSatisfiable)

Reply:
Added